### PR TITLE
build: update cypress version to 4.0.1

### PR DESCRIPTION
**Pull request by DlgSHi**
### Checklist

- [ ] Release notes (Conventional Commits) 
- [ ] Unit tests
- [x] Cypress automation tests
- [ ] Storybook added or updated
- [ ] Theme support
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/features/accessibility/accessibility.feature [hunk 1]
```diff
@@ -7,78 +7,98 @@ Feature: Accessibility tests
...
-      | component           |
-      | alert               |
-      | sidebar             |
+      | component |
+      | alert     |
+      | sidebar   |
+
+  @accessibility
+  Scenario: Component button as sibling
+    When I open "button" component page as sibling in no iframe
```
Comment (KatarzynaQA) [inline cypress/features/accessibility/accessibility.feature]: Given - When -Then

File: cypress/features/accessibility/accessibility.feature [hunk 2]
```diff
@@ -7,78 +7,98 @@ Feature: Accessibility tests
...
+      And I open Accessibility Tab
+    Then "button" component has no violations in Accessibility section
+
+  @accessibility
+  Scenario Outline: Component <component> page and open preview
```
Comment (KatarzynaQA) [inline cypress/features/accessibility/accessibility.feature]:
```suggestion
  Scenario Outline: Component <component> page
```

File: cypress/features/accessibility/accessibility.feature [hunk 3]
```diff
@@ -7,78 +7,98 @@ Feature: Accessibility tests
...
+    When I open "<component>" component page
```
Comment (KatarzynaQA) [inline cypress/features/accessibility/accessibility.feature]: Given - When -Then

File: cypress/features/accessibility/accessibility.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/accessibility/accessibility.feature]: updated

File: cypress/features/accessibility/accessibility.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/features/accessibility/accessibility.feature]: updated

File: cypress/features/accessibility/accessibility.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/accessibility/accessibility.feature]: updated

**Pull request merged**